task-launcher/src/tasks/trog/timeline.js

```diff
@@ -22,6 +22,7 @@ export default function buildTROGTimeline(config, mediaAssets) {
     responseAllowed: true,
     promptAboveButtons: true,
     task: config.task,
+    showPrompt: false
```

Comment:
Can you make this a part of the layoutConfig parameter that you see above? But this is a great approach. We should not tie configs to visual logic.

Reply:
Sure! I think I did this correctly in my last commit, but let me know if this wasn't what you had in mind and I can change it